aips/objects.md

```diff
@@ -152,15 +143,14 @@ There is an early reference implementation in the following PR: https://github.c
 
 Open areas for discussion include:
 
-- Whether or not to offer the ability for objects to have new resources added after creation? The current proposal has `ExtendRef` that can call `move_to`
-- Should `Object` be single owner and should that single owner be placed in the `Object` itself. As a result do we need `amount` in `OwnerRef` or `OwnerRef` at all. Note, we’d likely want some form of read-only Ref, so that if an object owns another object, it can be determined by reading on-chain only data.
 - Should addition and deletion of resources count the number of resources within an object to ensure that deletion is complete?
-- Should ownership be aggregable?
-- Should we block this on the availability of `BucketTable` or something similar for tracking ownership
+- How explicit should this AIP be about any of the abilities? Should there be a BaseObject for trivial usage that dictates an owner-centric set of policies.
+- Deletion could be complete, but then we need to ensure two objects can never be stored at the same address. An objected created over another risks corrupting storage due to overlapping event streams.
```

Comment:
for sure; what if deletion did:
1. delete the object (and everything in it)
2. create a new empty object in the same spot
3. set `deleted` to `true`, which would make it immutable

Reply:
No storage refund... We could offer two models